lib/chroma-docker-image-ecr-kms-deployment-cdk-stack.ts

```diff
@@ -27,11 +28,14 @@ export class ChromaDockerImageEcrDeploymentCdkStack extends cdk.Stack {
         ecrRepository.addLifecycleRule({ maxImageAge: cdk.Duration.days(7), rulePriority: 1, tagStatus: ecr.TagStatus.UNTAGGED }); // delete images older than 7 days
         ecrRepository.addLifecycleRule({ maxImageCount: 4, rulePriority: 2, tagStatus: ecr.TagStatus.ANY }); // keep last 4 images
 
-        // Copy from docker registry to ECR.
-        new ecrDeploy.ECRDeployment(this, `${props.appName}-${props.environment}-ECRDeployment`, {
-            src: new ecrDeploy.DockerImageName('chromadb/chroma:latest'),
-            dest: new ecrDeploy.DockerImageName(`${ecrRepository.repositoryUri}:${props.imageVersion}`),
-        });
+        const deployImageVersions = props.imageVersion === LATEST_IMAGE_VERSION? [props.imageVersion] : [props.imageVersion, LATEST_IMAGE_VERSION];
```

Comment:
**Suggestion:** To improve performance, consider using a `Set` to store `deployImageVersions` to avoid potential duplicates.
> 🌓  **Medium Priority** 
 > [category: **`performance`**, importance: **`5`**]
> **Reason:** Using a `Set` could prevent duplicates and slightly improve performance, but the impact is minor as the array size is typically small. 

```suggestion
        const deployImageVersions = new Set(props.imageVersion === LATEST_IMAGE_VERSION ? [props.imageVersion] : [props.imageVersion, LATEST_IMAGE_VERSION]);
```